### PR TITLE
Getblobs

### PR DESCRIPTION
**Pull request by MangyCoyote**

### CODE REVIEW

File: src/azmi-main/Operations.cs [hunk 1]
```diff
@@ -97,11 +101,41 @@ public string getBlob(string blobURL, string filePath, string identity = null, b
             }
         }
 
+        public List<string> getBlobs(string containerUri, string directory, string prefix = null, string identity = null)
+        {
+            containerUri.TrimEnd('/');
+            List<string> blobsListing = this.listBlobs(containerUri, identity, prefix);
+            if (blobsListing == null)
```
Comment (iricigor) [inline src/azmi-main/Operations.cs]: 👍

File: src/azmi-commandline/azmi-commandline.cs [hunk 2]
```diff
@@ -33,9 +33,8 @@ static RootCommand ConfigureArguments()
             };
 
             var shared_verboseOption = new Option(new String[] { "--verbose", "-v" })
-            {
-                Argument = new Argument<bool>("bool"),
```
[the remark this answers is not in the document]
Comment (MangyCoyote) [reply src/azmi-commandline/azmi-commandline.cs]: Simply I do not see a reason for this row(s). "bool" won't be displayed in a help as opposite to "string".

File: src/azmi-main/Operations.cs [hunk 3]
```diff
@@ -97,11 +101,41 @@ public string getBlob(string blobURL, string filePath, string identity = null, b
             }
         }
 
+        public List<string> getBlobs(string containerUri, string directory, string prefix = null, string identity = null)
+        {
+            containerUri.TrimEnd('/');
```
[the remark this answers is not in the document]
Comment (MangyCoyote) [reply src/azmi-main/Operations.cs]: Was somehow sure it modifies string in place. Seems it is not true:
https://www.dotnetperls.com/trimend

File: src/azmi-commandline/azmi-commandline.cs [hunk 4]
```diff
@@ -79,29 +79,58 @@ static RootCommand ConfigureArguments()
...
 
             rootCommand.AddCommand(getBlobCommand);
 
+            //
+            // getblobs
+            //
+
+            var getBlobsCommand = new Command("getblobs", "Downloads blobs from container to local directory.");
+
+            var getBlobs_containerOption = new Option(new String[] { "--container", "-c" })
```
[the remark this answers is not in the document]
Comment (MangyCoyote) [reply src/azmi-commandline/azmi-commandline.cs]: Let's discuss this in cleaning sprint.

**Pull request merged**